### DISCUSSION TITLE
How to send video files in response?

### DISCUSSION THREAD

**Discussion by robsteneha** [fastapi/fastapi#7249] [Questions]
My issue is that I want to take a video as an input perform object detection on it and return the resulting video in the response.
I am using UploadFile for taking the input video file but I don't know how to send back a video as a response.

**Comment by tiangolo** [comment]
I assume you solved it, thanks for closing the issue :+1:

**Comment by theoll** [comment]
@robsteneha How did you handle it?

**Comment by ayushitripathi23** [comment]
Hi @ tiangolo My issue is that I want to take a video as an input perform object detection on it and return the resulting video in the response.
I am using UploadFile for taking the input video file but I don't know how to send back a video as a response

**Comment by JarroVGIT** [comment]
This is a copy paste of the above issue. What have you tried, where are you stuck?
